frontend/src/components/pages/GlobalCodeReviewDashboard/FilterArea/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import Presentation from "./Presentation";
+
+function filterByFilterGroup({ allFilters, filter }) {
+  if (filter) {
+    return allFilters.filter((card) =>
+      card.toLowerCase().includes(filter.toLowerCase())
+    );
+  }
+  return allFilters;
```

Comment:
maybe this can be a util file? would it be useful for other things?

Reply:
It could be, I think I will need to change of the names to be more general so that it's obvious you can reuse it in other places

Reply:
It could be, I think I will need to change of the names to be more general so that it's obvious you can reuse it in other places